boards/arm/twr_ke18f/twr_ke18f.dts

```diff
@@ -125,6 +125,7 @@
 &sim {
 	clkout-source = <1>;
 	clkout-divider = <0>;
+	pinctrl-0 = <&CLKOUT_PTE10>;
```

Comment:
This will be likely moved under a SIM clkout node once I have converted the NXP Kinetis SIM/KE1xF SIM to use a proper clock dts definition.